src/ontology/mondo-edit.obo

```diff
@@ -579736,6 +579738,15 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-18T18:06:40Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7858" xsd:anyURI
 
+[Term]
+id: MONDO:1040015
+name: infectious disease with sepsis
+def: "Life-threatening organ dysfunction caused by a dysreguated host response to an infection." [PMID:26903338]
```

Comment:
@yshwetar
- there is a typo: "dysreguated" should be "dysregulated"
- as it currently is, the definition says the term's concept is an "organ dysfunction", which is not correct, it a "disease". The definition should start with "genus". In this case, it should be "An infectious disease characterized by ..."